add notice about eip2537 to arbos40 callisto (#2500)
OffchainLabs/arbitrum-docs: docs/run-arbitrum-node/arbos-releases/arbos40.mdx

#### [EIP-2537: Precompile for BLS12-381 curve operations](https://eips.ethereum.org/EIPS/eip-2537)

:::caution

> This EIP and its specified precompiles are part of ArbOS 40 Callisto but are _not_ enabled. The enabling of these precompiles will be proposed in a future ArbOS release.

Author: ```suggestion
This EIP and its specified precompiles are part of ArbOS 40 Callisto but are _not_ enabled. However, these precompiles will be proposed for inclusion in the next ArbOS release.
```
